src/app/components/Card/EditionCard.jsx

```diff
@@ -198,16 +200,30 @@ export default class EditionCard {
       <Link
         className="edition-card__card-info-link"
         to={{ pathname: '/read-online', search: `?url=${formatUrl(selectedLink.url, process.env.APP_ENV)}`, state: { work } }}
+        onClick={() => gaUtils.trackGeneralEvent('Read Online', editionItem.source, work.title, '')}
       >
         Read Online
       </Link>
     );
   }
 
-  static getDownloadLink(editionItem) {
+  static getDownloadLink(work, editionItem) {
     if (!editionItem || !editionItem.links) return undefined;
     const selectedLink = editionItem.links.find(link => link.download);
-    return selectedLink && selectedLink.url ? formatUrl(selectedLink.url, process.env.APP_ENV) : undefined;
+
+    if (selectedLink && selectedLink.url) {
+      return (
+        <a
+          className="edition-card__card-info-link"
+          href={`${formatUrl(selectedLink.url, process.env.APP_ENV)}`}
+          onClick={() => gaUtils.trackGeneralEvent('Download', editionItem.source, work.title, '')}
+        >
+          Download
+        </a>
+      );
+    }
+
+    return undefined;
```

Comment:
this shouldn't be reachable

Reply:
Just removed it in the merge to update the `formatUrl` calls